examples/components/CCFRAUD_VERTICAL/traininsilo/aml_comm.py

```diff
@@ -233,22 +234,16 @@ def _send(self, data, destination, flag, ok_flag):
                     )
             except Exception as e:
                 logger.exception(e)
-                tries += 1
+                retries += 1
+                self._stats["send_retries"] += 1
```

Comment:
Shouldn't this be `send_retries_cnt`? (here and througout, also for recv, also in other file)